### DISCUSSION TITLE
Add Wiki files to main .github folder in repository

### DISCUSSION THREAD

**Discussion by Josh65-2201** [community/community#38473] [Repositories]
When working with multiple people or on a major change which requires wiki updates there's no easy way to upload and apply all at once, since they seem to have their own private section on a repository.

It would be great to have the wiki pages in ```.github/wiki/pageName.md```
This can also allow easy backup/copying

**Comment by airtower-luna** [comment]
The wiki is a separate repository, which you can clone locally, see [Adding or editing wiki pages locally](https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages#adding-or-editing-wiki-pages-locally). You can clone/fetch, commit, and push as usual. No PRs unfortunately, though.

You could even set up an Actions workflow to mirror files with the main repo, if you like.

**Reply by Josh65-2201** [reply]
Whilst it nice it can be copied it quite annoying and makes no sense to have it that way since everything else is directly on the repository.
